commerce-cloud-code/app_yourshophere/cartridge/partials/cart/addtocart.js

```diff
@@ -28,23 +35,23 @@ exports.createModel = function createModel(options) {
 exports.template = (model) => `<dialog open>
 <article>
     <a href="#close"
-        aria-label="Close"
+        aria-label="${model.resources.close}"
         class="close"
         data-target="modal-example"
         onClick="this.closest('dialog').outerHTML=''">
     </a>
-  <h3>Your product has been added to cart</h3>
+  <h3>${model.resources.title}</h3>
   <p>
     ${model.quantity}x - ${model.text} - ${model.price}
   </p>
   <footer>
-    <a href="" role="button" class="secondary" onClick="this.closest('dialog').outerHTML=''">Continue Shopping</a>
+    <a href="" role="button" class="secondary" onClick="this.closest('dialog').outerHTML=''">${model.resources.continueShopping}</a>
```

Comment:
I falsely thought we have no custom JS in the main thread :) I might think about some css trickery